framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -230,6 +271,30 @@
 	    }
 	    return list;
 	  }
+
+
+private static List<File> searchFolders(File root, String regex) {
+          List<File> list = new ArrayList<File>();
+          Pattern pattern = Pattern.compile(regex + "$");
+          Matcher matcher = null;
+          if (!root.isFile()) {
+            matcher = pattern.matcher(root.getAbsolutePath());
+            if (matcher.find()) {
+              list.add(root);
+              return list;
+            }
+	    else{
+          	for (File file : root.listFiles()) {
+              		if (!file.getName().equals("datasources")) {
+                		list.addAll(searchFolders(file, regex));
+              		}
```

Comment:
Can you check if the spacing matches rest of the code? may be you'll need to use spaces instead of tabs